src/components/Dropdown/__tests__/Dropdown.test.tsx

```diff
@@ -148,34 +153,39 @@ describe('Dropdown', () => {
 
   it('deselects option when clicking on them twice when dropdown is multi', async () => {
     const { container, getByTestId, getByText, getAllByText } = render(
-      <Dropdown onSelect={mockedSelectHandler} multi name="choosePokemon" options={pokeOptions} />,
+      <Dropdown
+        onSelect={mockedSelectHandler}
+        multi
+        dataTestId="choosePokemon"
+        options={pokeOptions}
+      />,
     );
 
     act(() => {
-      fireEvent.focus(getByTestId('choosePokemon-container'));
+      fireEvent.focus(getByTestId('choosePokemon-dropdown-button'));
```

Comment:
This test-id change ends up being a breaking change, is there a reason we need to make this change? I'd rather keep defaults the same wherever possible

Reply:
It's not that the test-id changed, we're just not focusing the container anymore.